druid/src/widget/container.rs

```diff
@@ -16,18 +16,18 @@
 
 use crate::shell::kurbo::{Point, Rect, RoundedRect, Size};
 use crate::{
-    BoxConstraints, Data, Env, Event, EventCtx, LayoutCtx, LifeCycle, LifeCycleCtx, PaintBrush,
-    PaintCtx, RenderContext, UpdateCtx, Widget, WidgetPod,
+    BoxConstraints, Color, Data, Env, Event, EventCtx, KeyOrValue, LayoutCtx, LifeCycle,
+    LifeCycleCtx, PaintCtx, RenderContext, UpdateCtx, Widget, WidgetPod,
 };
 
 struct BorderStyle {
-    width: f64,
-    brush: PaintBrush,
+    width: KeyOrValue<f64>,
+    brush: KeyOrValue<Color>,
 }
 
 /// A widget that provides simple visual styling options to a child.
 pub struct Container<T> {
-    background: Option<PaintBrush>,
+    background: Option<KeyOrValue<Color>>,
```

Comment:
I think this isn't quite right, but is useful as an illustration.

We'll still likely want a `PaintBrush` enum in druid proper that includes `KeyOrValue<Color>` but perhaps some other things as well.